.github/README.md

```diff
@@ -6,107 +6,139 @@
 [![license](https://img.shields.io/github/license/Minestom/Minestom?style=for-the-badge&color=b2204c)](../LICENSE)
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=for-the-badge)](https://github.com/RichardLitt/standard-readme)  
 [![javadocs](https://img.shields.io/badge/documentation-javadocs-4d7a97?style=for-the-badge)](https://javadoc.minestom.net)
-[![wiki](https://img.shields.io/badge/documentation-wiki-74aad6?style=for-the-badge)](https://wiki.minestom.net/)
+[![wiki](https://img.shields.io/badge/documentation-wiki-74aad6?style=for-the-badge)](https://minestom.net/docs/introduction)
 [![discord-banner](https://img.shields.io/discord/706185253441634317?label=discord&style=for-the-badge&color=7289da)](https://discord.gg/pkFRvqB)
 
 Minestom is an open-source library that enables developers to create their own Minecraft server software, without any code from Mojang.
 
-The main difference between Mojang's vanilla server and a minestom-based server, is that ours does not contain any features by default!
-However, we have a complete API which is designed to allow you to make anything possible, with ease.
+Minestom does not include any vanilla features by default, however we have an extensive API which allows you to code any feature with ease.
 
-This is a developer API not meant to be used by end-users. Replacing Bukkit/Forge/Sponge with this **will not work** since we do not implement any of their APIs.
+This is a library for developers, and thus it is not meant to be used by the average server owner. Replacing a Bukkit, Forge, or Fabric server with this **will not work**, because we do not implement their APIs.
```

Comment:
"the average server owner" is pretty subjective. Maybe something like this?
`This is a server building library, not a server jar. You will need to write and compile code to create your own server jar using minestom.`

Reply:
Sounds too similar to the installation paragraph right below it